### PR TITLE
Add usage example to getting started [13416]

### PR DESCRIPTION
**Pull request by juanlofer-eprosima**
Merge after #105

### CODE REVIEW

File: docs/rst/getting_started/getting_started.rst [hunk 1]
```diff
@@ -58,8 +54,9 @@ This allows to create a non limited and highly scalable decentralized and distri
 .. figure:: /rst/figures/ddsrouter_overview_wan.png
 
 
+#################
 Usage Description
-=================
+#################
```
Comment (jparisu) [inline docs/rst/getting_started/getting_started.rst:59]: I would use the older section level, or separate them in different pages

File: docs/rst/getting_started/usage_example.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Apart from selecting on which topics we wish to send/receive data, we must configure as well the participants that will
+ultimately perform communication. Each router instance will contain a :ref:`simple <user_manual_participants_simple>`
+and a :ref:`wan <user_manual_participants_wan>` participants. In brief, the simple participants will be in charge of
+locally communicating with the corresponding ShapesDemo application, while the wan participants will be the ones
+bridging the connection between the two DDS networks.
+
+.. figure:: /rst/figures/shapesdemo_detailed.png
+
+The only configuration required for simple participants is the DDS ``domain`` identifier.
```
Comment (jparisu) [inline docs/rst/getting_started/usage_example.rst:71]: NIT: Add subsections for LAN and WAN

File: docs/rst/getting_started/usage_example.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+After setting up both routers, communication between the two ShapesDemo instances should have been established so the
+square shape is now visible in the subscriber’s panel.
+
+.. figure:: /rst/figures/shapesdemo_square.png
+
+|ddsrouter| supports the dynamic addition/deletion of topics at runtime (see
+:ref:`Reload Topics <user_manual_user_interface_reload_topics>`). Let us test this feature by adding the circle topic to
+the allowlist of both routers, and also by removing the square topic (removing this topic from one of the routers’
+allowlist will suffice).
```
Comment (jparisu) [inline docs/rst/getting_started/usage_example.rst:114]:
```suggestion
|ddsrouter| supports the dynamic addition/deletion of topics at runtime (see
:ref:`Reload Topics <user_manual_user_interface_reload_topics>`). Let us test this feature by adding the circle topic to
the allowlist of both routers
Also, by removing the square topic (removing this topic from one of the routers allowlist will suffice) the square data will stop reaching the subscriber..
```

File: docs/rst/getting_started/usage_example.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+.. code-block:: yaml
+
+    allowlist:
+    [
+        {name: "Circle", type: "ShapeType", keyed: true}
+    ]
+
+After applying these changes, the square should no longer be visible on the subscriber’s side (actually visible but
+frozen), while the circle should.
```
Comment (jparisu) [inline docs/rst/getting_started/usage_example.rst:124]:
```suggestion
After applying these changes, the square should no longer be updated in the subscriber's side (visible but
frozen), while the circle should.
```

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/rst/getting_started/getting_started.rst (hunk 1 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/getting_started.rst:59]: :+1:

File: docs/rst/getting_started/usage_example.rst (hunk 2 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/usage_example.rst:71]: The example is valid both for LAN and WAN scenarios, with some small differences such as the one pointed out in the initial note.

File: docs/resources/getting_started/client-ddsrouter.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  type: wan
+  id: 1
+  connection-addresses:
+  [
+    {
+      id: 0,
+      addresses:
+      [
+        {
+          ip: "192.168.1.8",
```
[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/resources/getting_started/client-ddsrouter.yaml:22]: I changed the initial note a bit so hopefully it is more clear that the given configurations are for the LAN case.

File: docs/rst/getting_started/usage_example.rst (hunk 3 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/usage_example.rst:114]: :+1:

File: docs/rst/getting_started/usage_example.rst (hunk 4 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/usage_example.rst:124]: :+1:

**Review by jparisu — COMMENTED** [review]
LGTM
Please rebase

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**